src/compiler/zx_calculus.jl

```diff
@@ -0,0 +1,167 @@
+using ZXCalculus
+using ZXCalculus: qubit_loc, clifford_simplification
+import IRTools: IR
```

Comment:
don't use `import`, if you need to overload a function write it explicitly `IRTools.IR`